.code-samples.meilisearch.yaml

```diff
@@ -463,9 +509,9 @@ phrase_search_1: |-
   curl -X POST 'http://localhost:7700/indexes/movies/search' \
   -H 'Content-Type: application/json' \
   --data-binary '{ "q": "\"african american\" horror" }'
-authentication_header_1: |-
+authorization_header_1: |-
   curl -X GET 'http://127.0.0.1:7700/keys' \
-    -H 'X-Meili-API-Key: masterKey'
+    -H 'Authorization: Bearer masterKey'
```

Comment:
Wouldn't you be consistent with the other way you use to authenticate with the master key?
Seen below:
```
-H 'Authorization: Bearer [your_master_key]'
```